coderd/database/queries/licenses.sql

```diff
@@ -0,0 +1,9 @@
+-- name: InsertLicense :one
```

Comment:
It seems weird to me that multiple licenses could be supported. Why allow multiple?